tsl/test/sql/bgw_custom.sql

```diff
@@ -78,40 +79,45 @@ SELECT delete_job(NULL);
 SELECT delete_job(-1);
 \set ON_ERROR_STOP 1
 
--- We keep job 1000 for some additional checks.
-SELECT delete_job(1001);
+-- We keep job 1001 for some additional checks.
 SELECT delete_job(1002);
 SELECT delete_job(1003);
 SELECT delete_job(1004);
+SELECT delete_job(1005);
 
 -- check jobs got removed
-SELECT count(*) FROM timescaledb_information.jobs WHERE job_id >= 1001;
+SELECT count(*) FROM timescaledb_information.jobs WHERE job_id >= 1002;
 
 \c :TEST_DBNAME :ROLE_SUPERUSER
 
 \set ON_ERROR_STOP 0
 -- test bad input
 SELECT alter_job(NULL, if_exists => false);
 SELECT alter_job(-1, if_exists => false);
+SELECT alter_job(1001, job_name => 'this_is_a_really_really_really_long_application_name_to_overflow');
```

Comment:
I think one test that we can add here is manually changing the job_id (update the job table) let's say to 2147483647 (max for int4) and then try to alter the job and make sure will work if the name fit with big job_id and fail if the name is bigger than the NAMEDATALEN.

Reply:
Done.